spec.md

```diff
@@ -1500,7 +1500,7 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 
 A Controller Plugin MUST implement this RPC call if it has `CREATE_DELETE_SNAPSHOT` capability.
 This RPC will be called by the CO to delete a snapshot.
-If successful, the storage space associated with the snapshot MUST be released and all the data in the snapshot SHALL NOT be accessible anymore.
+If successful, the storage space associated with the snapshot MAY or MAY NOT be released depending on the reclaim or retention policy specified on the storage system.
```

Comment:
Same note regarding Kubernetes centric phrasing.  I understand that's not the intent but was thinking we could be more agnostic here and still get the point across.  Thoughts?